### ISSUE TITLE
Option to sort installed extensions by state (turned on or off)

### ISSUE THREAD

**Issue by forteller** [mjakeman/extension-manager#97]
Some times I might not remember the name of the extension I want to change some settings of. So I have to scroll trough my list and look at the names. Then it would be very useful if I could sort the list such that all turned on extensions show first, instead of there being a mix of on and off extensions.

Hope that makes sense. Thanks for this fantastic application!

**Comment by mjakeman** [comment]
Hi, thanks for the interest!

From a technical standpoint, this is easy  to implement. My concern is how the UI would work for this - I want to avoid having too many options or toggles on the screen as it would impact the simplicity of the app. My initial thinking is to add a "Display Enabled First" check option in the menu.

Out of curiosity, how many extensions do you have installed?

It would help me better understand your use case especially in regards to #67 (I'm curious to see how that would scale with a large number of extensions).

**Comment by forteller** [comment]
Thanks for the reply! At the moment I have 33 user-installed extensions installed (14 enabled) and 13 system extensions (0 enabled).

Regarding the UI I agree it's important that things aren't cluttered. I'd be happy to have this in the hamburger menu. I also have another idea, but I totally understand if that's not working for you (or maybe not possible in Gnome):

![Screenshot from 2022-02-09 00-49-10](https://user-images.githubusercontent.com/343561/153095296-6194facf-c222-4b5a-b94e-5cda6ba6638b.png)

Just a very bad attempt at illustrating the point. A test only dropdown in that general area. It should be blue of course.

Again, no problem if this doesn't fit. I don't mind much where it is :)

**Comment by mjakeman** [comment]
> Thanks for the reply! At the moment I have 33 user-installed extensions installed (14 enabled) and 13 system extensions (0 enabled).

That's a lot more than I have :0

I've got maybe 10 total? Will definitely keep in mind with the redesign, as having too many local results could make unified search a bit cumbersome.

> I also have another idea, but I totally understand if that's not working for you (or maybe not possible in Gnome):

It definitely looks nicer. From a usability perspective however, it conveys to me that it would only sort the user-installed extensions (rather than the system ones too). So it's probably unworkable in practice, since we'd want to avoid a redundant second dropdown below.

I've added this to the list for 0.3 - Shouldn't be too hard hopefully
